src/gwt/src/org/rstudio/studio/client/common/filetypes/ScriptFileType.java

```diff
@@ -78,6 +79,25 @@ public String getScriptInterpreter()
          result.add(commands.sourceActiveDocument());
          result.add(commands.sourceActiveDocumentWithEcho());
       }
+
+      if (canExecuteCode())
```

Comment:
I'm guessing that the goal here to remove commands which were previously added here:

https://github.com/rstudio/rstudio/blob/54cd3abcfc58837b433464c793fe9b03a87f0bb4/src/gwt/src/org/rstudio/studio/client/common/filetypes/TextFileType.java#L314-L327

I'm worried that it will be hard to keep these in sync (i.e. if we add another code command, we're going to have to remember to go back to this new list and take it out again so it doesn't wind up inappropriately enabled in script files). Maybe we need a different flag for executing script, or some declarative way to establish this so that we're not likely to drift out of sync?